api/utils/validators.js

```diff
@@ -76,6 +76,10 @@ function isEmptyOrUrl(value) {
   }
 }
 
+const validBasicAuthUsername = s => /^[a-zA-Z0-9_-]{4,15}$/.test(s);
+
+const validBasicAuthPassword = s => /^[a-zA-Z0-9_\-$!@]{6,25}$/.test(s);
```

Comment:
How were the regexes determined?

Reply:
@davemcorwin - init just keeping it simple. do you have suggestions to augment it?

Reply:
Just wondering where it came from. It feels too restrictive to me and wondering if there are reasons for that. For example, my email address wouldn't work as a username.

Reply:
upated based on https://owasp.org/www-community/OWASP_Validation_Regex_Repository safetext and password